scripts/index.js

```diff
@@ -0,0 +1,707 @@
+/* global window document */
```

Comment:
this file is mostly a copy of `old-index.js` (previously `index-demo-page.js`). I'll point out new pieces below.